libs/monorepo-utils/src/circleci.ts

```diff
@@ -155,6 +157,10 @@ ${jobIds.map((jobId) => `      - ${jobId}`).join('\n')}
 commands:
   checkout-and-install:
     description: Get the code and install dependencies.
+    parameters:
+      path:
+        type: string
+        default: '.'
```

Comment:
We could edit the parameterization to default to installing all packages for the repo instead of just root packages.